### PR TITLE
Add enable_mesos_input_plugin config option

### PR DESCRIPTION
**Pull request by gracedo**
https://jira.mesosphere.com/browse/DCOS-47301
Adds the enable_mesos_input_plugin config option.

Terraform Core PR: dcos-terraform/terraform-template-dcos-core#2
DCOS PR: dcos/dcos#4222

### CODE REVIEW

File: main.tf
Comment (geekbass) [inline main.tf]: Needs set back to this source prior to merge.

File: main.tf
Comment (geekbass) [inline main.tf]: Will also need to set the version back prior to merge.

**Review by geekbass — CHANGES_REQUESTED** [review]
Code LGTM but need to ensure that we set back the source and version in the main.tf for dcos-bootstrap module. The checks will fail for this until we cut a new release for the underlying module which was merged this AM.

Adding @fatz here for second pair of eyes.

File: main.tf
Comment (gracedo) [reply main.tf]: will do

**Review by bernadinm — APPROVED** [review]
LGTM!

**Comment by bernadinm** [comment]
When Jenkins is back up, lets rerun the job before merging if possible.

**Review by geekbass — APPROVED** [review]

**Comment by bernadinm** [comment]
manually ran the test. Here is the output, good to merge:
```
$ terraform fmt
$ terraform validate --check-variables=false
$ docker run -ti -v$(pwd):/a --entrypoint /usr/bin/terraform-docs fatz/terraform-jenkins md /a | tr -d '\r' > README.md
$ git status
On branch gracedo/enable_mesos_input_plugin_config_option
Your branch is up to date with 'origin/gracedo/enable_mesos_input_plugin_config_option'.```

**Pull request merged**
